File: TransCoder_tokenized_test_set_functions/java/FUNCTION_COPY_STRING_ITERATIVE_RECURSIVE_1.java
static void myCopy ( char s1 [ ], char s2 [ ], int index ) {
  s2 [ index ] = s1 [ index ];
  if ( index == s1 . length - 1 ) {
    return;
  }
  myCopy ( s1, s2, index + 1 );
}

File: results/py/FUNCTION_COPY_STRING_ITERATIVE_RECURSIVE_1.cpp
int f_filled ( string s1 , string s2 , int index ) {
  s2 [ index ] = s1 [ index ] ;
  ;
  if ( ( index == s1 . length ( ) - 1 ) || ( index == s2 . length ( ) - 1 ) ) return index ;
  ;
  return index + 1 ;
}
